winterfell/webapp/src/js/app.js

```diff
@@ -4,56 +4,126 @@
 'use strict';
 
 var app = angular.module('vetafiApp', [
-  'ngRoute',
   'angular-click-outside',
-  'ngDialog',
   'signature',
   'formly',
   'formlyBootstrap',
   'angular-momentjs',
-  'ui.bootstrap'
+  'ui.bootstrap',
+  'ui.router'
 ]);
 
 /**
  * Configure routes
  */
-app.config(function ($routeProvider) {
-  $routeProvider.when("/", {
-    templateUrl: "templates/home.html"
-  }).when("/faq", {
-    templateUrl: "templates/faq.html"
-  })
-
-    // Profile Pages
-    .when("/profile", {
-      templateUrl: "templates/profile.html"
-    }).when("/profile/:subPage", {
-      templateUrl: "templates/profile.html"
-    })
-
-    // File Claims Pages
-    .when("/tos", {
-      templateUrl: "templates/tos.html"
-    }).when("/claim/start", {
-      templateUrl: "templates/claimStart.html"
-    }).when("/claim/:claimId/select-forms", {
-      templateUrl: "templates/claimSelectForms.html"
-    }).when("/claim/confirm", {
-      templateUrl: "templates/claimConfirm.html"
-    }).when("/claim/submitted", {
-      templateUrl: "templates/claimSubmitted.html"
-    }).when("claim/form/:formId", {
-      template: "<div>Some Form</div>" // replace to claim/form/:formId when it's ready
-    }).when("/claim/:claimId", {
-      templateUrl: "templates/claimView.html"
-    })
-
-    .when("/claim/:claimId/form/:formId", {
-      templateUrl: "templates/form.html"
-    })
-
-    // Otherwise
-    .otherwise({
-      redirectTo: '/'
-    });
-});
+app.config(['$stateProvider', '$urlRouterProvider', function ($stateProvider, $urlRouterProvider) {
+
+  $stateProvider.state({
+    name: 'root',
+    url: '/app',
```

Comment:
Yeah so basically this is the "parent" state, and it will be at the root of our app, and all pages under it will have certain things in common (in this case, all pages under this will try to resolve the user)

We dont have to call it `/app`, we could just have it be `/`, but I thought it would be useful if we want to have another angular page tree, its alot easier to distinguish but giving this a real path